src/test/java/hudson/security/LDAPEmbeddedTest.java

```diff
@@ -245,7 +245,35 @@ public void groupLookup() throws Exception {
         GroupDetails groupDetails = r.jenkins.getSecurityRealm().loadGroupByGroupname("HMS Bounty");
         assertThat(groupDetails.getDisplayName(), is("HMS Bounty"));
         assertThat(groupDetails.getName(), is("HMS Bounty"));
-        assertThat("LDAP security realm does not support group member query", groupDetails.getMembers(), nullValue());
+        assertThat("LDAP security realm does not fetch group members by default", groupDetails.getMembers(), nullValue());
+    }
+
+    @Test
+    @LDAPSchema(ldif = "sevenSeas", id = "sevenSeas", dn = "o=sevenSeas")
+    public void groupLookupWithMembers() throws Exception {
+        r.jenkins.setSecurityRealm(new LDAPSecurityRealm(
+                ads.getUrl(),
+                null,
+                null,
+                null,
+                null,
+                null,
+                new FromGroupSearchLDAPGroupMembershipStrategy(null),
+                "uid=admin,ou=system",
+                Secret.fromString("pass"),
+                false,
+                false,
+                new LDAPSecurityRealm.CacheConfiguration(100, 1000),
+                new LDAPSecurityRealm.EnvironmentProperty[0],
+                "cn",
+                null,
+                IdStrategy.CASE_INSENSITIVE,
+                IdStrategy.CASE_INSENSITIVE)
+        );
+        GroupDetails groupDetails = r.jenkins.getSecurityRealm().loadGroupByGroupname("HMS Bounty", true);
```

Comment:
Probably useful to also test what happens when `false` is passed (or the parameter is omitted), unless an existing test is already doing this.

Reply:
[LDAPEmbeddedTest#groupLookup](https://github.com/dwnusbaum/ldap-plugin/blob/38176866f1f38b486d572dff54975ff89a35a769/src/test/java/hudson/security/LDAPEmbeddedTest.java#L245) omits the parameter.